htdocs/adherents/type.php

```diff
@@ -214,7 +228,7 @@
 {
 	//dol_fiche_head('');
 
-	$sql = "SELECT d.rowid, d.libelle as label, d.subscription, d.vote";
+	$sql = "SELECT d.rowid, d.libelle as label, d.subscription, d.vote, d.welcome, d.price, d.vote, d.automatic, d.family";
```

Comment:
@ptibogxiv 
"d.vote" is defined twice

Reply:
@hregis thank you i change it immediately